extensions/applicationinsights-perfmarkmeasure-js/scripts/listCdnVersions.ps1

```diff
@@ -573,20 +573,20 @@ Function ListVersions(
 
                 if ($paths.ContainsKey($thePath) -ne $true) {
                     $paths[$thePath]  = $true
-                    $value = "{0,-28}" -f $thePath
+                    $value = "{0,-35}" -f $thePath
                     $pathList = "$pathList$value  "
                 } else {
                     $paths[$thePath] = ($paths[$thePath] + 1)
                 }
             }
 
             foreach ($thePath in $paths.Keys | Sort-Object) {
-                Log $("  - {1,-48} ({0})" -f $paths[$thePath],$thePath)
+                Log $("  - {1,-55} ({0})" -f $paths[$thePath],$thePath)
             }
 
-            Log $("v{0,-12} ({1,2})  -  {2}" -f $key,$($fileList.Count),$pathList.Trim())
+            Log $("v{0,-16} ({1,2})  -  {2}" -f $key,$($fileList.Count),$pathList.Trim())
```

Comment:
Just want to confirm these numbers are updated correctly... you must counted but not sure why previous numbers changes from `8`-`16` and here from `12`-`16`.

Reply:
Yep, this is just formatting for the output, I needed to add more space because of the pre-release tags